cpp/src/io/json/json_column.cu

```diff
@@ -572,7 +572,10 @@ std::pair<std::unique_ptr<column>, std::vector<column_name_info>> device_json_co
     CUDF_EXPECTS(json_col.validity.size() >= bitmask_allocation_size_bytes(json_col.num_rows),
                  "valid_count is too small");
     auto null_count =
-      cudf::detail::null_count(json_col.validity.data(), 0, json_col.num_rows, stream);
+      json_col.type == json_col_t::StringColumn
+        // Skipped because calculated again in parse_data
```

Comment:
Could this be fixed? I'd like for us to no rely on `UNKNOWN_NULL_COUNT` going forward.

Reply:
`UNKNOWN_NULL_COUNT` is not used in this case. For `StringColumn`, it uses only uses null mask.

Reply:
What should go here if `UNKNOWN_NULL_COUNT` is removed in the future.